mlflow/utils/cli_args.py

```diff
@@ -40,6 +40,12 @@
 PORT = click.option("--port", "-p", default=5000,
                     help="The port to listen on (default: 5000).")
 
+ACTIVATE_SCHEDULER = click.option("--activate-scheduler", is_flag=True,
+                                  help="If specified, will activate periodic jobs ran by the"
+                                       " server. You also need to specify a configuration file"
+                                       " using --scheduler-configuration to chose which job"
+                                       " you wish to activate and their configuration.")
```

Comment:
typo choose
Sentence is unclear. 

What about to merge the 2 options. If an conf file is specified then the scheduler is enabled ?